app/src/ui/changes/changed-file.tsx

```diff
@@ -112,7 +117,13 @@ export class ChangedFile extends React.Component<IChangedFileProps, void> {
         action: () => this.props.onRevealInFileManager(this.props.path),
         enabled: this.props.status !== AppFileStatus.Deleted,
       },
+      {
+        label: 'Open file',
```

Comment:
Let's also make this a little clearer by labeling it as "Open with External Editor."